src/frontend/components/router-tree/router-tree.ts

```diff
@@ -52,7 +52,7 @@ export class RouterTree {
     };
 
     const nodes = this.tree(d3.hierarchy(
-      root.children.length > 1 ? root : root.children[0], d => d.children));
+      root.children.length === 0 || root.children.length > 1 ? root : root.children[0], d => d.children));
```

Comment:
As discuss two things are happening, one is the short-circuit operation, that will result in a boolean `true` being used, so the ternary operator will never get evaluated. The other is more of a readability edit.

Only on this expression `root.children.length === 0` being `false` will the ternary operator get evaluated.